site/en/guide/autodiff.ipynb

```diff
@@ -491,7 +491,7 @@
         "id": "ISkXuY7YzIcS"
       },
       "source": [
-        "By default, the resources held by a `GradientTape` are released as soon as `GradientTape.gradient()` method is called. To compute multiple gradients over the same computation, create a `persistent` gradient tape. This allows multiple calls to the `gradient()` method as resources are released when the tape object is garbage collected. For example:"
+        "By default, the resources held by a `GradientTape` are released as soon as `GradientTape.gradient` method is called. To compute multiple gradients over the same computation, create a `persistent` gradient tape. This allows multiple calls to the `gradient()` method as resources are released when the tape object is garbage collected. For example:"
```

Comment:
```suggestion
        "By default, the resources held by a `GradientTape` are released as soon as the `GradientTape.gradient` method is called. To compute multiple gradients over the same computation, create a `persistent` gradient tape. This allows multiple calls to the `gradient()` method as resources are released when the tape object is garbage collected. For example:"
```

Reply:
Is `persistent` an arg that's supposed to be in code blocks? If not, please remove those backticks

Reply:
Thanks @lamberta 👍 Yup, `persistent` is an arg indeed.

```diff
tf.GradientTape(
    persistent=False, watch_accessed_variables=True
)
```

Source: https://www.tensorflow.org/api_docs/python/tf/GradientTape

Reply:
Thanks @anirudh161. I like how you rephrased it: `Create a gradient tape with persistent=True`. I kept the "If you need to do this, do that" structure, since it's an option and I wanted to have the user involved with "you". Hopefully, this is alright:

```diff
By default, the resources held by a `GradientTape` are 
released as soon as the `GradientTape.gradient` method is called.
+ To compute multiple gradients over the same computation, 
+ create a gradient tape with `persistent=True`. 
...[explanation]...
For example:
```

cc @lamberta 

This will be reflected in PR https://github.com/tensorflow/docs/pull/1769 (the autodiff guide)